package.json

```diff
@@ -7,7 +7,7 @@
   "engines": {
     "node": "8.4.0",
     "npm": "5.3.0",
-    "yarn": "1.0.1"
+    "yarn": "1.*"
```

Comment:
I moved this since this prevents `yarn install` from running (sans Docker/CI) which might throw some people off.

Reply:
Cool thanks - I forgot to loosen that dependency.  It's pinned in the Dockerfile so should be consistent on CI at least.